Bad_Practices/Task_Estimation/TEBP.md

```diff
@@ -0,0 +1,13 @@
+## Mistakes to avoid when performing task estimation 
+
+____
+
+### 1. Excesive Optismism
+Providing an estimation of time for how long it will take to complete tasks should be optimsitic with a certain level of confidence, however sometimes this optimism can lead to many issues. Developers tend to forget about many of the trivial parts of development process, meetings, absences, technical issues and much more.Care should be taken to not create estimates thinking in terms of _"Coding Hours"_.
```

Comment:
```suggestion
Providing an estimation of time for how long it will take to complete tasks should be optimistic with a certain level of confidence, however sometimes this optimism can lead to many issues. Developers tend to forget about many of the trivial parts of development process, meetings, absences, technical issues and much more. Care should be taken to not create estimates thinking in terms of _"Coding Hours"_.
```